code/src/utils/is-error-with-status/isErrorWithStatus.test.ts

```diff
@@ -20,7 +20,7 @@ describe("Test isErrorWithStatus", () => {
   });
 
   test("should return false if status is not a number", () => {
-    const res = isErrorWithStatus("400");
+    const res = isErrorWithStatus({ status: "400" });
```

Comment:
Maybe it would be better to use one approach in variablr naming and use "result" instead of 'res'?

Reply:
Fixed